velox/vector/tests/VariantToVectorTest.cpp

```diff
@@ -36,7 +36,7 @@ class VariantToVectorTest : public testing::Test, public test::VectorTestBase {
     EXPECT_EQ(vector->type()->toString(), type->toString());
     EXPECT_TRUE(vector->isNullAt(0));
 
-    auto copy = vectorToVariant(vector, 0);
```

Comment:
Let's add some tests with 'index' other than 0.

Reply:
Makes sense. Is that okay to do it in follow-up PR ?

Reply:
That's fine.